laterpay/application/Controller/Admin/Appearance.php

```diff
@@ -163,13 +180,15 @@ public static function process_ajax_requests( LaterPay_Core_Event $event ) {
             case 'purchase_button_position':
                 $purchase_button_pos_form = new LaterPay_Form_PurchaseButtonPosition( $_POST );
 
-                if ( ! $purchase_button_pos_form->is_valid( ) ) {
-                    $event->set_result(
-                        array(
-                            'success' => false,
-                            'message' => __( 'An error occurred when trying to save your settings. Please try again.', 'laterpay' ),
-                        )
+                try {
+                    $purchase_button_pos_form->validate( $_POST );
```

Comment:
use full variable names, pos -> position, to avoid things like 'dogs' -> 'date of graduate students'